project_or_component_name/src/main.rs

```diff
@@ -0,0 +1,24 @@
+// TODO: Update the name of the method loaded by the prover. E.g., if the method is `multiply`, replace `METHOD_NAME_ID` with `MULTIPLY_ID` and replace `METHOD_NAME_PATH` with `MULTIPLY_PATH`
+use methods::{METHOD_NAME_ID, METHOD_NAME_PATH};
+use risc0_zkvm::host::Prover;
+// use risc0_zkvm::serde::{from_slice, to_vec};
+
+fn main() {
+    // Make the prover.
+    let method_code = std::fs::read(METHOD_NAME_PATH)
+        .expect("Method code should be present at the specified path; did you use the correct *_PATH constant?");
+    let mut prover = Prover::new(&method_code, METHOD_NAME_ID)
+        .expect("Prover should be constructed from valid method source code and corresponding method ID");
+
+    // TODO: Implement communication with the guest here
```

Comment:
Hmm... I'm worried that if I find myself reading this code and am not aware of the associated markdown tutorial you're planning to pair with it, I'll be out at sea

Reply:
I wonder whether we could provide some more clear signal for how readers should find out how to implement this TODO.